electrum/transaction.py

```diff
@@ -88,7 +88,16 @@ class MissingTxInputAmount(Exception):
     pass
 
 
-SIGHASH_ALL = 1
+class Sighash(IntFlag):
```

Comment:
I recommend `IntEnum` instead of `IntFlag` here.
Flag is very confusing here as only ANYONECANPAY is a true flag, there other values are not...
E.g. `ALL | NONE == SINGLE` <- but this does not make sense semantically.